src/include/platform/CHIPDeviceConfig.h

```diff
@@ -647,28 +647,26 @@
  * CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING_INTERVAL_MIN
  *
  * The minimum interval (in units of 0.625ms) at which the device will send BLE advertisements while
- * in the extended advertising mode. The minimum interval shall not be smaller than the default value
- * and should not be equal to the CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING_INTERVAL_MAX.
+ * in the extended advertising mode. The minimum interval shall not be smaller than the default value.
  *
- * Defaults to 1920 (1200 ms).
+ * Defaults to 2056 (1285 ms).
  */
-#define CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING_INTERVAL_MIN 1920
+#define CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING_INTERVAL_MIN 2056
```

Comment:
Merge gate comment: merge only after https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/8965 is merged.

Reply:
Referenced pull request has been merged.